pants.remote-cache.toml

```diff
@@ -0,0 +1,25 @@
+# Enable remote caching by running `./pants --pants-config-files=pants.remote-cache.toml`.
+#
+# You must also set:
+#
+#  - `remote_store_store`
+#  - `remote_oauth_bearer_token_path`
+#  - `remote_ca_certs_path`, which is usually "/etc/ssl/certs/ca-certificates.crt" on Linux and
+#      "/usr/local/etc/openssl/cert.pem" on macOS.
+#
+# You may want to set those values in an `.envrc` file and use `direnv` (https://direnv.net/) to automatically set the
+# values.
+
+[GLOBAL]
+remote_cache_read = true
+remote_cache_write = true
+
+remote_store_initial_timeout = 250
+remote_store_timeout_multiplier = 1.5
+remote_store_maximum_timeout = 5000
```

Comment:
Should these be adjustments to the default values? If not, explaining why we set them to non-defaults would be good.

Reply:
I have no idea. Blindly copied from Tom's PR.